sdk/trace/src/main/java/io/opentelemetry/sdk/trace/data/StatusData.java

```diff
@@ -68,8 +70,14 @@ default boolean isUnset() {
    *
    * @return {@code true} if this {@code Status} is OK or UNSET.
    */
-  // TODO: Consider to remove this in a future PR. Avoid too many changes in the initial PR.
   default boolean isOk() {
```

Comment:
Also considering inverting this to `isFailure` so it uses a word that doesn't match one of the code names.

Reply:
Maybe better to deprecate this as well....

Reply:
In a separate PR if you decire